app/pb_kits/playbook/pb_timeline/_item.jsx

```diff
@@ -0,0 +1,54 @@
+/* @flow */
+
+import React from 'react'
+import classnames from 'classnames'
+import { buildCss } from '../utilities/props'
+import { DateStacked, IconCircle } from '..'
+
+type ItemProps = {
+  className?: String,
+  children?: Array<React.ReactChild>,
+  date?: Date,
+  icon?: String,
+  iconColor?: String,
+  lineStyle?: String,
```

Comment:
string is lowercase on flow, and `children` is usually `Node` AFAIK:

```suggestion
  className?: string,
  children?: Array<React.Node>,
  date?: Date,
  icon?: string,
  iconColor?: string,
  lineStyle?: string,
```

Reply:
Good call!